app/components/post-search-results-affiliate-item.js

```diff
@@ -8,12 +8,18 @@ export default Component.extend({
 
   classNames: ['post-search-results-affiliate-item'],
 
-  getUnitLink: computed('unit', 'isInContent', function () {
+  getUnitLink: computed('affiliateUnit', 'isInContent', function () {
     if (this.isInContent && this.affiliateUnit.links && this.affiliateUnit.links.article) {
       this.affiliateUnit.link = this.affiliateUnit.links.article;
     } else if (!this.isInContent && this.affiliateUnit.links && this.affiliateUnit.links.search) {
       this.affiliateUnit.link = this.affiliateUnit.links.search;
     }
+
+    if (this.affiliateUnit.campaign === 'ddb') {
```

Comment:
Looks like a hack - is there any way that we can move those params to slots config? (it doesn't have to be right now)

Reply:
discussed offline